src/ng2-datetime-picker.directive.ts

```diff
@@ -263,6 +265,7 @@ export class Ng2DatetimePickerDirective implements OnInit, OnChanges {
   };
 
   hideDatetimePicker = (event?): any => {
+    console.log('what');
```

Comment:
Whoops, sorry! That was from when I was trying to find which event was closing the datepicker. :)